moon/controller.py

```diff
@@ -331,7 +331,7 @@ def try_step_around(self):
         # recovered enough at this point to switch to another driver (in case you see cubesat while doing the 3m drive or the final turn)
         self.bus.publish('driving_recovery', False)
 
-        self.go_straight(3.0, timeout=timedelta(seconds=20))
```

Comment:
is this function related to R3 only?

Reply:
No, try_step_around is available for any Round to use as they see fit. The idea is if you hit something going forward, you try to side-step it, ie go a little back, turn 90, go forward a little, turn -90 and continue where you left off. Presumably obstacle avoidance will improve eventually so this is only a rough first attempt in that direction.